.github/workflows/encryption-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Encryption Tests 
+
+on:
+  push: 
+    branches: ['master']
+  pull_request:
+    branches: [ 'master' ]
+  workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
+
+jobs:
+  run-tests:
+    permissions:
+      # required for all workflows
+      security-events: write
+      id-token: write
+      contents: write
+    runs-on: ubuntu-latest
+    name: Encryption tests
+    env:
+      FORCE_COLOR: true
+    steps:
+      - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+      - name: Setup node
+        uses: actions/setup-node@0a44ba7841725637a19e28fa30b79a866c81b0a6 # v4.0.4
+        with:
+          node-version: 22
```

Comment:
I switched the GH action to run-on Node v22 instead of latest because of a dependency installation bug ([NODE-6641](https://jira.mongodb.org/browse/NODE-6641)) that was stopping us from installing the necessary `mongo-client-encryption` dependency. 

The local tests still work fine for me.